components/Layout.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import Link from 'next/link';
+import {
+  HomeIcon,
+  SearchIcon,
+  BellIcon,
+  MailIcon,
+  DocumentTextIcon,
+  BookmarkIcon,
+  UserIcon,
+  DotsCircleHorizontalIcon,
+  PencilIcon,
+} from '@heroicons/react/outline';
+import IconTwitter from '../components/IconTwitter';
+import IconButton from '../components/IconButton';
+import Button from '../components/Button';
+import Modal from '../components/Modal';
+import Tweet from '../components/Tweet';
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const arrayObjects = [
```

Comment:
Move the constants to another file. It takes too much space inside a component file